src/ws-notification/notification.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Body, Controller, Post } from "@nestjs/common";
+import { ApiDto } from "./dto/api.dto";
+import { SubscriptionDto } from "./dto/subscription.dto";
+import { NotificationService } from "./notification.service";
+
+@Controller("ws-notify")
+export class NotificationController{
+    constructor(
+        private readonly NotificationService: NotificationService
+    ){}
+
+    @Post('/subscription-event')
+    async sendNewSubscription(
+        @Body() dto: SubscriptionDto
+    ){
+        return await this.NotificationService.sendNewSubscriptionEvent(dto)
```

Comment:
What should be the return value when this endpoint is hit? I tested with random uuids and the response was undefined.  I don't know if I did the right thing, so can you walk me through this?

Reply:
Did you set up a frontend. All you have to do is fill up the apiId, userId, developerId and execute. It should emit the event which a frontend can listen to and return the apiId and userId. I have a frontend setup already, i can send it if you want.